src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/DoubleBookingVisitTest.kt

```diff
@@ -79,4 +84,25 @@ class DoubleBookingVisitTest : IntegrationTestBase() {
       lastVisitDto = currentVisitDto
     }
   }
+
+  @Test
+  fun `When booking a visit a capacity exception roles back to uncompleted booking`() {
```

Comment:
sorry minor typo - rolls back to incomplete

Reply:
Well you know me, and my good spelling :-)